docker/bin/install-etcd.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -ex
+
+ETCD_ARCHIVE_URL=https://github.com/coreos/etcd/releases/download/v0.4.6/etcd-v0.4.6-linux-amd64.tar.gz
```

Comment:
might be nice to rewrite as defaulting to an external source, so that we could more easily test different etcd releases:
```sh
ETCD_ARCHIVE_URL=${ETCD_ARCHIVE_URL:-https://github.com/coreos/etcd/releases/download/v0.4.6/etcd-v0.4.6-linux-amd64.tar.gz}
```